docs/nidigital/class.rst

```diff
@@ -3483,7 +3483,7 @@ set_attribute_vi_session
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: set_attribute_vi_session(attribute)
+    .. py:method:: set_attribute_vi_session(attribute, value)
```

Comment:
please open issue to either stop generating this or make it private

Reply:
Made it private.

You don't usually like unrelated changes in a PR